test/vector.js

```diff
@@ -44,6 +46,11 @@ tape('vector', function (t) {
     Buffer.from(mfHex, 'hex'),
     Buffer.from(sf2Hex, 'hex').toString('base64')
   )
+  const sf3Keys = deriveFeedKeyFromSeed(
+    Buffer.from(sf3Hex, 'hex'),
+    'a pre existing feed',
+    'gabbygrove-v1'
+  )
```

Comment:
Maybe one of the sources of headaches is this. `deriveFeedKeyFromSeed` doesn't support `gabbygrove-v1`, so it will treat this `sf3Keys` as classic. This might *accidentally* work, but it's still accidentally. Which is why I recommended avoiding gabbygrove because we don't want to give ourselves headaches

Reply:
I agree that this might trip us up in the future. I could add explicit handling of `gabbygrove-v1` format to `deriveFeedKeyFromSeed` in `ssb-meta-feeds` (it looks like it would be a near-exact replica of the current code for `bendybutt-v1`)?

Reply:
Yes, in fact it would just an addition of ` || format === 'gabbygrove-v1` in the `if`

Reply:
Support added: https://github.com/ssb-ngi-pointer/ssb-meta-feeds/pull/40